src/org/labkey/test/components/glassLibrary/grids/Pager.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2019 LabKey Corporation. All rights reserved. No portion of this work may be reproduced in
+ * any form or by any electronic or mechanical means without written permission from LabKey Corporation.
+ */
+package org.labkey.test.components.glassLibrary.grids;
```

Comment:
I've been putting off making a new package/migrating everything into it, largely because I don't have a clear vision for what the new destination should look like and I didn't want to make the changes and then be told I did it wrong.  I'll just do my best for now.